content/soarian/dstu2/general-clinical/condition.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Condition | Soarian DSTU 2 API
+---
+ 
+# Condition
+
+* TOC
+{:toc}
+
+## Overview
+
+For the Condition resource query, the API returns a consolidated list of conditions that providers have charted in the Soarian Problem List.  
+
+Problems returned may be in active, inactive, or resolved state.  Problems marked erroneous are not returned.  Both the problems that will be returned and the problem detail for each, depend on client configuration and charting practice. Problems returned include free text problems and those structured problems configured in the clinical care category.  The narrative response returns data as charted and may be more robust than the structured response. 
+
+
+## Terminology Bindings  
+
+<%= terminology_table(:soarian_condition, :dstu2) %>
+
+## Extensions
+The content of these extensions must be visible to the user, and recorded with any storage or transmit activity along with the query/response date/time, as it represents the best representation of what clinicians recorded for the patient at the time of query.  
+
+
+| Name | Condition | Description |
```

Comment:
Same comment as on diagnostic report. Should be able to remove this and reference a common section on Overview.